portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,59 +31,123 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
I think this doc can be updated since it's no long example content and you've achieved your TODO :)